python/Ganga/test/GPI/Credentials/TestCredentials.py

```diff
@@ -14,13 +14,10 @@
 def test_voms_proxy_life_cycle(gpi):
     from Ganga.GPI import VomsProxy, credential_store
 
-    try:
-        _ = credential_store[VomsProxy()]
-        # There is no auto-detection of a VomsProxy built into Ganga atm
-        # requesting it is expected to fail.
-        assert False
-    except KeyError:
-        pass
+    # check that we clear the credential store before we do anything else
+    credential_store.clear()
+    
+    assert len(credential_store) == 0
```

Comment:
Depending on what plugin is running depends on if there is a credential at start up or not. I couldn't get an easy way sorting this out so just thought it best to clear it to start with.